backend/config/initializers/cors.rb

```diff
@@ -7,7 +7,12 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins ENV['BRANCH_ORIGIN'] || ENV['FRONTEND_ORIGIN'] || 'http://localhost:81'
+    origins [
+      ENV['BRANCH_ORIGIN'],
+      ENV['FRONTEND_ORIGIN'],
+      'http://localhost:81',
+      'http://localhost:5173'
```

Comment:
```suggestion
      'http://localhost:81', # vue front用
      'http://localhost:5173' # react front用
```

readmeも修正しておくと良いかもしれないですね